listenbrainz_spark/utils.py

```diff
@@ -7,16 +7,28 @@
 
 import listenbrainz_spark
 from listenbrainz_spark.stats import run_query
-from listenbrainz_spark import stats, config, path
+from listenbrainz_spark import stats, config, path, schema
 from listenbrainz_spark import hdfs_connection
 from listenbrainz_spark.exceptions import FileNotSavedException, ViewNotRegisteredException, PathNotFoundException, \
-    FileNotFetchedException, HDFSDirectoryNotDeletedException
-
+    FileNotFetchedException, DataFrameNotCreatedException, DataFrameNotAppendedException, HDFSDirectoryNotDeletedException
 from flask import current_app
 from hdfs.util import HdfsError
 from brainzutils.flask import CustomFlask
 from pyspark.sql.utils import AnalysisException
 
+def append(df, dest_path):
```

Comment:
at first glance these functions don't do much, but I see how they will improve the readability of the code. +1